spec/lib/granite/context_proxy_spec.rb

```diff
@@ -59,21 +59,6 @@ def self.hash
         expect(klass.proxy_context).to eq context
       end
     end
-
-    it 'keeps context in thread safe way' do
```

Comment:
This spec sometimes gets stuck while waiting for `thread.join`. I've tried debugging it but couldn't find any issue. It seems ruby randomly decides that it wants to sleep the threads and we can't really control it.
I've tried alternatives but they haven't worked either. Either ruby runs threads one by one, which means we can't cause a failure or randomly puts threads to sleep which means `join` gets stuck.

There's already https://github.com/toptal/granite/blob/master/spec/lib/granite/context_proxy_spec.rb#L79 which checks that we store current context in thread safe way, it will have to be enough.